tests/ipld.rs

```diff
@@ -24,11 +24,12 @@ mod std_tests {
 
     use std::iter::FromIterator;
 
+    #[allow(clippy::useless_format)]
     #[test]
     fn serde() {
         let tuple_struct = TupleStruct(format!("test"), -60, 3000);
 
-        let tuple = (format!("hello"), -50.0040957, -12.094635556478);
```

Comment:
I'm just curious, was this a lint (the number change)?

Reply:
Probably because rust can't accurately represent it?

Reply:
(that's a total guess)

Reply:
Yes, the lint was: https://rust-lang.github.io/rust-clippy/master/index.html#excessive_precision